chronos/timer.nim

```diff
@@ -31,8 +31,7 @@ const asyncTimer* {.strdefine.} = "mono"
 
 when defined(windows):
   when asyncTimer == "system":
-    proc fastEpochTime*(): uint64 {.
-         inline, deprecated: "Use Moment.now()".} =
+    proc fastEpochTime*(): uint64 {.inline, deprecated: "Use Moment.now()".} =
```

Comment:
unrelated formatting changes are best left out of feature PR:s (notably, chronos does not use nph and it's thus best to follow the local style of the file)